app/routes/location.php

```diff
@@ -23,3 +23,4 @@
 );
 Route::get('location-reporting/{location_id}/{question_category_id}', 'LocationReportController@show2');
 Route::post('api/set-search-radius', 'LocationSearchController@setSearchRadius');
+Route::post('add-suggestion', 'LocationReportController@addSuggestion');
```

Comment:
Since this route will return JSON instead of HTML, prefix with "api/" like "api/add-suggestion".  All other routes connected with JSON responses in AccessLocator have the "api/" prefix so this will make things more consistent.